File: Integraupt-Backend/BackendAdReserva/src/Models/Facultad.php
<?php

namespace App\Models;

class Facultad {
    public ?int $id;
    public ?string $nombre;
    public ?string $abreviatura;

    public function __construct(?int $id = null, ?string $nombre = null, ?string $abreviatura = null) {
        $this->id = $id;
        $this->nombre = $nombre;
        $this->abreviatura = $abreviatura;
    }
}

File: integraupt-web/Integraupt-Backend/BackendSanciones/src/main/java/com/Sanciones/model/Facultad.java
package com.Sanciones.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "facultad")
public class Facultad {

    @Id
    @Column(name = "IdFacultad")
    private Long id;

    @Column(name = "Nombre", nullable = false)
    private String nombre;

    @Column(name = "Abreviatura")
    private String abreviatura;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getAbreviatura() {
        return abreviatura;
    }

    public void setAbreviatura(String abreviatura) {
        this.abreviatura = abreviatura;
    }
}